tests/test_random_variables/test_normal.py

```diff
@@ -234,52 +239,47 @@ def test_symmetric_samples(self):
     def test_indexing(self):
         """ Indexing with Python integers yields a univariate normal distribution. """
         for mean, cov in self.normal_params:
-            dist = prob.Normal(mean=mean, cov=cov)
+            rv = rvs.Normal(mean=mean, cov=cov)
 
-            if isinstance(
-                dist,
-                (
-                    prob.distributions.normal._UnivariateNormal,
-                    prob.distributions.normal._OperatorvariateNormal,  # TODO: Implement slicing on linear operators
-                ),
+            if isinstance(rv.mean, linops.LinearOperator) or isinstance(
+                rv.cov, linops.LinearOperator
             ):
+                # TODO: Implement slicing on linear operators
```

Comment:
Can you create an issue for this missing test and reference it in this TODO?

Reply:
I instead implemented (advanced) indexing, slicing, and masking for `Normal` random variables using `todense` and removed the comment altogether :)

Reply:
I will still open an issue though